lib/private/Files/View.php

```diff
@@ -490,6 +493,14 @@ public function readfilePart($path, $from, $to) {
 		return false;
 	}
 
+
+	private function checkConnectionStatus(): void {
+		$connectionStatus = \connection_status();
+		if ($connectionStatus !== 0) {
```

Comment:
I think it can be more explicit that way:
```suggestion
		if (\connection_aborted()) {
```

Reply:
Yeah but it gives less information, in the debug log the connectionStatus is stored, which may help debug weird edge cases related to this later.

Reply:
We can keep the `connection_status()` for the exception.

At first I was bothered by the "arbitrary" value of zero.
So, maybe we can write it like this instead:
```suggestion
		if ($connectionStatus !== CONNECTION_NORMAL) {
```